### ISSUE TITLE
Subscritions to channel with filters works wrong in @logux/client TestServer

### ISSUE THREAD

**Issue by AleksandrSl** [logux/logux#93]
Subscriptions are added and removed by `action.channel` 
```
if (action.type === 'logux/subscribe') {
    if (!this.subscriptions[action.channel]) {
      this.subscriptions[action.channel] = {}
    }
}
...
```
But for subscription with filters `action.channel` is always the same. We could either consider adding serialized filters to the subscription key or choose different way of storing subscriptions.

**Comment by ai** [comment]
Seems like we need to have:

```js
this.subscriptions[action.channel] = (Filter|true)[]
```

Can I ask you to send PR?

**Comment by AleksandrSl** [comment]
`this.subscriptions[action.channel][nodeId] = true` we use this map to store subscribed nodes.
We could use
```
this.subcriptions[action.channel][nodeId] = Filter | true []
```
or 
```
this.subcriptions[action.channel] = [Filter, [nodeId]] | [null, nodeId] []
```

or we could hash filter and just use it as a key, I like this variant cause finding all the subscribers would be easy
```
this.subcriptions[action.channel] = Record<filterHash, nodeId[]>
```

but we need some reliable hash for js objects

**Comment by ai** [comment]
Let's use `this.subcriptions[action.channel][nodeId] = Filter | true []` to avoid breaking changes

**Comment by euaaaio** [comment]
Looks like we can close this issue. It has been fixed in logux/server#141
